.travis.yml

```diff
@@ -1,18 +1,29 @@
-sudo: false
-
 language: rust
-
-rust:
-  - stable
-  - nightly
+cache: cargo
 
 matrix:
-  allow_failures:
-    - rust: nightly
-
-before_script:
-  - rustup component add clippy rustfmt
+  include:
```

Comment:
Are you sure this actually improves things? Often, the setup costs end up being higher than the actual runtime?

Reply:
Yes, the improvement is that it is no clear which components are needed in which step and getting a nice build matrix.

>  Ran for 3 min 29 sec 

This is with a cold cache. I don't think speed is an issue for now :)

Reply:
I personally feel this is overkill insofar we do not perform any involved tests beyond Clippy and the crate does not have unit tests at all.

Reply:
> we do not perform any involved tests

[Huh?](https://github.com/bbqsrc/cucumber-rust/tree/e43c3612e1fa725a7dc10a3f0642296180f9dab6/tests) But well, then this is your chance to start writing tests for a testing library without having to add it CI afterwards because it's already there when you merge this 😄